README.md

```diff
@@ -55,6 +55,24 @@ pnpm exec biome --help
 
 To check types, you can run the `pnpm typecheck` command as other commands ignore types.
 
+## Tests
+
+The unit tests are written with [node:test](https://nodejs.org/api/test.html) and [node:assert](https://nodejs.org/api/assert.html).
+
+The unit tests can be run with the following command:
+
+```shell
+pnpm test
+```
+
+The end-to-end tests are written with [playwright](https://playwright.dev/).
+The tests are in `apps/class-solid/tests/*.spec.ts` and can be run with the following command:
+
+```shell
+cd ./apps/class-solid
+pnpm exec playwright test
```

Comment:
Does it make sense to add a `test` command in `package.json` for this?

Reply:
I also thought about it, but when running `pnpm test` from root the report command it prints is incorrect as it needs to be run with `--filter ./apps/class-solid` or`cd ./apps/class-solid`` before.

Also i want to distinguish between unit test vs end-to-end test